src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -15,7 +15,14 @@ After you've completed setting up a project in Sentry, Sentry will give you a va
 
 On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
 
-## Error Boundaries
+You can trigger your first event from your development environment by raising an exception somewhere within your application. A simple example of this would be rendering a button like so:
```

Comment:
```suggestion
You can trigger your first event from your development environment by raising an exception somewhere within your application. An example of this would be rendering a button like so:
```